### PR TITLE
Set background color early

### PR DESCRIPTION
**Pull request by marians**
This PR attempts to change the loading behaviour so that the final background color of the viewport is set as early as possible, preventing a white flash during loading.

Note that this uses the background color of the login page, which is darker than the app's normal background

## Before

image

Timing here based on webpagetest.org with "cable" as network bandwith (5 MBit/Sec).

Results: https://www.webpagetest.org/result/201016_DiYT_ae9f28b21731b39e2c4c287dfa1455ba/

## After

image

Results: https://www.webpagetest.org/result/201019_DiDF_5a79b726d23f0ad1c5c95755a8cebb8c/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -51,13 +56,12 @@
...
     <style>
-      body {
-        background: #234a61;
-      }
-
-      .loading footer {
-        display: none;
-      }
+    html, body {
+      background-color: #152531;
```
Comment (oponder) [inline src/index.html:60]: do we still need this here if we also moved it up now?

**Review by oponder — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (marians) [reply src/index.html:60]: Not sure, really. I left it in, since it could be that the CSS we load in-between resets the color again, no?

Comment (oponder) [reply src/index.html:60]: only if we explicitly have something in there that sets it to white or some other color again.. other wise, it's 'cascading style sheets', and it cascades into eachother.

**Review by oponder — APPROVED** [review]
approving so you can keep movin 👍 

I think the background-color at the bottom again shouldn't be needed, but not a big deal.

**Comment by marians** [comment]
Removed the redundant style definition and tested again. Works. Thanks!

**Pull request merged**